package.json

```diff
@@ -132,6 +132,7 @@
     "node-sass": "^4.14.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
+    "prettier-eslint": "^12.0.0",
```

Comment:
I'm not.. sure.. I installed it the same way I installed the other two.. I didn't notice anything using it specifically. But I guess it's hard for me to say since that was my first rodeo with it.